src/components/material/ReviewInfomedia.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {
+  InfomediaReview,
+  useGetInfomediaQuery
+} from "../../core/dbc-gateway/generated/graphql";
+import { useText } from "../../core/utils/text";
+import ReviewHearts from "./ReviewHearts";
+import ReviewMetadata from "./ReviewMetadata";
+
+export interface ReviewInfomediaProps {
+  review: InfomediaReview;
+}
+
+const ReviewInfomedia: React.FC<ReviewInfomediaProps> = ({ review }) => {
+  const { id } = review;
+  const { data } = useGetInfomediaQuery({
+    id
+  });
+  const t = useText();
+  return (
+    <li className="review text-small-caption">
+      {(review.author || review.date) && (
+        <ReviewMetadata author={review.author} date={review.date} />
```

Comment:
I think we should use date objects as properties whenever possible. It is much easier to do date calculations and test like that. So the date should be transformed into a date object before sending it to the ReviewMetadata.
You could create a helper function, eg. fromReviewDateToDateObject in a file called: src/components/material/helpers.ts.
Then you can use datejs to create a proper date object. Like Sine does here:
https://github.com/danskernesdigitalebibliotek/dpl-react/pull/55/files#diff-5796af082eb3b5376779c529940e1f83e906738f33dedb630a37d5205e4d7f89R20

Reply:
Thanks a lot for the extensive recommendation! 🏅 💯 I adjusted it so that we work with Date objects instead ;)